Comparing /root/./workspace/mcfongtw/flink/386270361/.m2/org/apache/flink/flink-streaming-java_2.11/1.1.4/flink-streaming-java_2.11-1.1.4.jar with /root/workspace/mcfongtw/flink/386270361/flink-streaming-java/target/flink-streaming-java_2.11-1.5-SNAPSHOT.jar:
===  UNCHANGED CLASS: PUBLIC org.apache.flink.streaming.api.datastream.AllWindowedStream  (not serializable)
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator apply(org.apache.flink.api.common.functions.ReduceFunction, org.apache.flink.streaming.api.functions.windowing.AllWindowFunction)
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator apply(org.apache.flink.api.common.functions.ReduceFunction, org.apache.flink.streaming.api.functions.windowing.AllWindowFunction, org.apache.flink.api.common.typeinfo.TypeInformation)
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator apply(java.lang.Object, org.apache.flink.api.common.functions.FoldFunction, org.apache.flink.streaming.api.functions.windowing.AllWindowFunction)
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator apply(java.lang.Object, org.apache.flink.api.common.functions.FoldFunction, org.apache.flink.streaming.api.functions.windowing.AllWindowFunction, org.apache.flink.api.common.typeinfo.TypeInformation)
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator fold(java.lang.Object, org.apache.flink.api.common.functions.FoldFunction)
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator fold(java.lang.Object, org.apache.flink.api.common.functions.FoldFunction, org.apache.flink.api.common.typeinfo.TypeInformation)
		+++  NEW ANNOTATION: java.lang.Deprecated
+++  NEW CLASS: PUBLIC(+) STATIC(+) org.apache.flink.streaming.api.datastream.CoGroupedStreams$UnionSerializerConfigSnapshot  (not serializable)
	+++  NEW SUPERCLASS: org.apache.flink.api.common.typeutils.CompositeTypeSerializerConfigSnapshot
	+++  NEW CONSTRUCTOR: PUBLIC(+) CoGroupedStreams$UnionSerializerConfigSnapshot()
	+++  NEW CONSTRUCTOR: PUBLIC(+) CoGroupedStreams$UnionSerializerConfigSnapshot(org.apache.flink.api.common.typeutils.TypeSerializer, org.apache.flink.api.common.typeutils.TypeSerializer)
	+++  NEW METHOD: PUBLIC(+) int getVersion()
===  UNCHANGED CLASS: PUBLIC org.apache.flink.streaming.api.datastream.KeyedStream  (not serializable)
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator fold(java.lang.Object, org.apache.flink.api.common.functions.FoldFunction)
		+++  NEW ANNOTATION: java.lang.Deprecated
***  MODIFIED CLASS: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator  (not serializable)
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.streaming.api.datastream.DataStream getSideOutput(org.apache.flink.util.OutputTag)
***  MODIFIED CLASS: PUBLIC org.apache.flink.streaming.api.datastream.WindowedStream  (not serializable)
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator apply(org.apache.flink.api.common.functions.ReduceFunction, org.apache.flink.streaming.api.functions.windowing.WindowFunction)
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator apply(org.apache.flink.api.common.functions.ReduceFunction, org.apache.flink.streaming.api.functions.windowing.WindowFunction, org.apache.flink.api.common.typeinfo.TypeInformation)
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator apply(java.lang.Object, org.apache.flink.api.common.functions.FoldFunction, org.apache.flink.streaming.api.functions.windowing.WindowFunction)
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator apply(java.lang.Object, org.apache.flink.api.common.functions.FoldFunction, org.apache.flink.streaming.api.functions.windowing.WindowFunction, org.apache.flink.api.common.typeinfo.TypeInformation)
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator fold(java.lang.Object, org.apache.flink.api.common.functions.FoldFunction)
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED METHOD: PUBLIC org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator fold(java.lang.Object, org.apache.flink.api.common.functions.FoldFunction, org.apache.flink.api.common.typeinfo.TypeInformation)
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator reduce(org.apache.flink.api.common.functions.ReduceFunction, org.apache.flink.streaming.api.functions.windowing.WindowFunction)
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator reduce(org.apache.flink.api.common.functions.ReduceFunction, org.apache.flink.streaming.api.functions.windowing.WindowFunction, org.apache.flink.api.common.typeinfo.TypeInformation)
***  MODIFIED CLASS: PUBLIC org.apache.flink.streaming.api.environment.CheckpointConfig  (compatible)
	+++  NEW METHOD: PUBLIC(+) boolean isFailOnCheckpointingErrors()
	+++  NEW METHOD: PUBLIC(+) void setFailOnCheckpointingErrors(boolean)
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.flink.streaming.api.environment.StreamExecutionEnvironment  (not serializable)
	+++  NEW METHOD: PUBLIC(+) java.util.List getCachedFiles()
	+++  NEW METHOD: PUBLIC(+) int getMaxParallelism()
	+++  NEW METHOD: PUBLIC(+) void registerCachedFile(java.lang.String, java.lang.String)
	+++  NEW METHOD: PUBLIC(+) void registerCachedFile(java.lang.String, java.lang.String, boolean)
	+++  NEW METHOD: PUBLIC(+) org.apache.flink.streaming.api.environment.StreamExecutionEnvironment setMaxParallelism(int)
***  MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.flink.streaming.util.serialization.DeserializationSchema  (default serialVersionUID changed)
	+++  NEW INTERFACE: org.apache.flink.api.common.serialization.DeserializationSchema
	+++  NEW ANNOTATION: java.lang.Deprecated
***  MODIFIED INTERFACE: PUBLIC ABSTRACT org.apache.flink.streaming.util.serialization.SerializationSchema  (default serialVersionUID changed)
	+++  NEW INTERFACE: org.apache.flink.api.common.serialization.SerializationSchema
	+++  NEW ANNOTATION: java.lang.Deprecated
***  MODIFIED CLASS: PUBLIC org.apache.flink.streaming.util.serialization.TypeInformationSerializationSchema  (superclass modified)
	***  MODIFIED SUPERCLASS: org.apache.flink.api.common.serialization.TypeInformationSerializationSchema (<- java.lang.Object)
	---  REMOVED METHOD: PUBLIC(-) java.lang.Object deserialize(byte[])
	---  REMOVED METHOD: PUBLIC(-) org.apache.flink.api.common.typeinfo.TypeInformation getProducedType()
	---  REMOVED METHOD: PUBLIC(-) boolean isEndOfStream(java.lang.Object)
	---  REMOVED METHOD: PUBLIC(-) byte[] serialize(java.lang.Object)
	+++  NEW ANNOTATION: java.lang.Deprecated
